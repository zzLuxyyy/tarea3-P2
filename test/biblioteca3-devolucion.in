crearBiblioteca
crearSocio 12345678 Jim Dear 10 03 2004
agregarSocioBiblioteca
crearSocio 23456789 Lucia Perez 10 03 2010
agregarSocioBiblioteca
crearFecha 05/06/1967
crearLibro 843760494 "Cien años de soledad" "Gabriel" "García Márquez" "La historia de la familia Buendía" 3
agregarLibroBiblioteca
crearFecha 15/08/2025
prestarLibroBiblioteca 12345678 843760494
imprimirReservasBiblioteca
crearFecha 30/08/2025
devolverLibroBiblioteca 12345678 843760494 15/08/2025
imprimirPrestamosBiblioteca
imprimirReservasBiblioteca
liberarBiblioteca
Fin