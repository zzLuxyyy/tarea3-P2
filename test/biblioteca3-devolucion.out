1>Biblioteca creada exitosamente.
2>Socio de ci 12345678 creado exitosamente.
3>Socio de CI 12345678 agregado a la biblioteca.
4>Socio de ci 23456789 creado exitosamente.
5>Socio de CI 23456789 agregado a la biblioteca.
6>7>8>Libro de ISBN 843760494 agregado a la biblioteca.
9>10>Préstamo del libro 843760494 al socio 12345678 registrado.
11>Cola de Reservas:
12>13>Devolución del libro 843760494 por el socio 12345678 registrada.
14>LDE Préstamos:
Préstamo de libro Cien años de soledad a Jim Dear.
15/8/2025
30/8/2025
15>Cola de Reservas:
16>Biblioteca liberada exitosamente.
17>Fin.
